apps/server/src/modules/board/controller/dto/element/file-element.response.ts

```diff
@@ -0,0 +1,35 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { ContentElementType } from '../../../types/content-elements.enum';
+import { TimestampsResponse } from '../timestamps.response';
+
+export class FileElementContent {
+	constructor({ description }: FileElementContent) {
+		this.description = description;
```

Comment:
Is this already decided by marco? What kind of metadata will be displayed alongside the file?